inc/az_ulib_ipc_api.h

```diff
@@ -124,9 +124,13 @@ static inline AZ_ULIB_RESULT az_ulib_ipc_publish(
 #endif /* AZ_ULIB_CONFIG_IPC_VALIDATE_CONTRACT */
 }
 
+#ifdef AZ_ULIB_CONFIG_IPC_UNPUBLISH
 /**
  * @brief   Unpublish an interface from the IPC.
  *
+ * @node    Please uncomment #AZ_ULIB_CONFIG_IPC_UNPUBLISH on az_ulib_config.h to enable this
```

Comment:
>Please uncomment #AZ_ULIB_CONFIG_IPC_UNPUBLISH on az_ulib_config.h to enable this [](start = 12, length = 81)

this seems to run contrary to what the instructions on `ulib_config.h` state. This seems to be included by default and the "note" should be instructions on how to remove it.
AKA: "To remove this api compile with `-DAZ_ULIB_REMOVE_UNPUBLISH`" or something like that

Reply:
sure.

---
In reply to: [342851883](https://github.com/Azure/azure-ulib-c/pull/47#discussion_r342851883) [](ancestors = 342851883)